### PR TITLE
[ML] APM latency correlations popover

### PR DESCRIPTION
**Pull request by lcawl**
## Summary

This PR is based on https://github.com/elastic/kibana/pull/99905

It adds an [EuiPopover](https://elastic.github.io/eui/#/layout/popover) similar to how they were added in [Lens](https://github.com/elastic/kibana/blob/master/x-pack/plugins/lens/public/indexpattern_datasource/help_popover.tsx) and contains content similar to what was originally written in the [Kibana guide](https://www.elastic.co/guide/en/kibana/master/correlations.html).

### Screenshots

image

### CODE REVIEW

**Comment by qn895** [comment]
Regarding 
> You can view the impact of other attributes by selecting them in the table

Right now our mechanism for showing is more hovering over the row than selecting... There's not a selection mechanism, but not sure how we would convey that 🤔

**Review by benwtrent — APPROVED** [review]

File: x-pack/plugins/apm/public/components/app/correlations/ml_latency_correlations_help_popover.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      <p>
+        <FormattedMessage
+          id="xpack.apm.correlations.latencyPopoverChartExplanation"
+          defaultMessage="The latency distribution chart visualizes the overall latency of the service and the attributes that are most likely responsible for slow transactions. You can view the impact of other attributes by selecting them in the table."
+        />
+      </p>
+      <p>
+        <FormattedMessage
+          id="xpack.apm.correlations.latencyPopoverTableExplanation"
+          defaultMessage="The table is sorted by correlation values, which are derived by using a combination of Pearson correlation coefficient (PCC) and Kolmogorov–Smirnov (K-S) tests and range from 0 to 1. Attributes with a high correlation are likely to contribute to increased latency."
```
Comment (tveasey) [inline x-pack/plugins/apm/public/components/app/correlations/ml_latency_correlations_help_popover.tsx]: This is almost the case:
> "which are derived by using a combination of Pearson correlation coefficient (PCC) and Kolmogorov–Smirnov (K-S) tests and range from 0 to 1."

However, it implies to me that score is some combination of both PCC and K-S when it is in fact just the PCC. The K-S test is used to exclude any fields for which there is little evidence that field actually results in slow transactions. I would consider changing it to something like:

"The table is sorted by Pearson correlation coefficient (PCC), which ranges from 0 to 1. A Kolmogorov–Smirnov (K-S) test is used to determine if the slow down is statistically significant. Attributes with a high correlation are likely to contribute to increased latency."

**Review by tveasey — APPROVED** [review]
Basically, LGTM but proposed one slight tweak.

File: x-pack/plugins/apm/public/components/app/correlations/ml_latency_correlations_help_popover.tsx (hunk 1 above)
Comment (lcawl) [reply x-pack/plugins/apm/public/components/app/correlations/ml_latency_correlations_help_popover.tsx]: Thanks, I think that's a lot clearer. I've updated the text in f802855b17893818df29cbb68bdd022d46ba7ea0

**Comment by lcawl** [comment]
> Regarding
> 
> > You can view the impact of other attributes by selecting them in the table
> 
> Right now our mechanism for showing is more hovering over the row than selecting... There's not a selection mechanism, but not sure how we would convey that 🤔

Thanks--I've updated that paragraph to talk about "hovering" instead in f802855b17893818df29cbb68bdd022d46ba7ea0

**Comment by lcawl** [comment]
This work is now moved to https://github.com/elastic/kibana/pull/103759 since the base branch was merged.

**Pull request closed without merging**